scripts/navigation.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+$('#homeNav').click(function(){
```

Comment:
Although this works, it's generally a better practice to use the on method to define event handlers and give the event name as an argument to 'on.' 
```
$('#homeNav').on('click', function() {
/// stuff
})
```